pkg/preflight/assets/host-preflights.yaml

```diff
@@ -184,6 +204,50 @@ spec:
               message: Port 2380 is open
           - warn:
               message: Unexpected port status
+    - tcpPortStatus:
+        checkName: "ETCD Health Server TCP Port Status"
+        collectorName: "ETCD Health Server TCP Port Status"
+        exclude: '{{ and .IsPrimary (not .IsUpgrade) | not }}'
+        outcomes:
+          - fail:
+              when: "connection-refused"
+              message: Connection to port 2381 was refused. This is likely to be a routing problem since this preflight configures a test server to listen on this port.
+          - warn:
+              when: "address-in-use"
+              message: Another process was already listening on port 2381.
+          - fail:
+              when: "connection-timeout"
+              message: Timed out connecting to port 2381. Check your firewall.
+          - fail:
+              when: "error"
+              message: Unexpected port status
+          - pass:
+              when: "connected"
+              message: Port 2381 is available
```

Comment:
what was the decision on this port? do we need to change it or document it?

Reply:
I meant to discuss that at standup. I left it as is because kubeadm configures the etcd pod to use this port as a health check.